flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/PlannerBase.scala

```diff
@@ -83,7 +83,9 @@ abstract class PlannerBase(
       config,
       functionCatalog,
       asRootSchema(new CatalogManagerCalciteSchema(catalogManager, isStreamingMode)),
-      getTraitDefs.toList
+      getTraitDefs.toList,
+      catalogManager.getBuiltinCatalogName,
+      catalogManager.getCatalog(catalogManager.getBuiltinCatalogName).get().getDefaultDatabase
```

Comment:
nit: shall we just introduce a `getBuiltInDatabaseName` and remove these chains here and at other locations?